configs/next/distros/redhat-8.mk

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Distro dependent basic definitions for RedHat Enterprise Server 8
+# =================================================================
+# This Makefile include contains the definitions for the distro being used
+# for the build process. It contains kernel version infos and distro specific
+# sanity checks.
+
+# Informs the distro supported power archs
+AT_SUPPORTED_ARCHS := ppc64le
+
+# You may force the BUILD_IGNORE_AT_COMPAT general condition by distro
+#BUILD_IGNORE_AT_COMPAT := yes
+
+# Kernel version to build toolchain against
+# - As this distro only supports one arch variation (ppc64le), there is no
+#   need to conditionally define these versions based on arch.
+AT_KERNEL := 4.18    # Current distro kernel version for runtime.
+# Previous distro kernel version for runtime-compat.
+AT_OLD_KERNEL :=
+
+# Inform the mainly supported distros
+AT_SUPPORTED_DISTROS := RHEL8
+
+# Inform the compatibility supported distros
+AT_COMPAT_DISTROS :=
+
+# Sign the repository and packages
+AT_SIGN_PKGS := yes
+AT_SIGN_REPO := yes
+AT_SIGN_PKGS_CROSS := no
+AT_SIGN_REPO_CROSS := yes
+
+# ID of GNUPG key used to sign our packages (main/compat)
+AT_GPG_KEYID  := 6976A827
+AT_GPG_KEYIDC := 6976A827
+AT_GPG_KEYIDL := 6976A827
+# ID of GNUPG key used to sign our repositories (main/compat)
+AT_GPG_REPO_KEYID  := 6976A827
+AT_GPG_REPO_KEYIDC := 3052930D
```

Comment:
I don't think this is necessary, based on the rest of the changes, but a perfectly correct value would have been 6976A827 too because RHEL 7 was also signed with that key.

Reply:
Fixed